packages/sdk/CHANGELOG.md

```diff
@@ -2,6 +2,25 @@
 
 - YYYY-MM-DD **BREAKING?** -- Who: description
 
+- 2021-09-20 -- [@chasefleming](https://github.com/chasefleming): Resolve asynchronous arguments with specific `resolveArgument` method over generic `resolve`
+
+### Example of resolveArgument for async args.
```

Comment:
a heading of `###` i believe will be picked up by the documentation stuff and be put in the sidebar. I would probably just make it bold.